<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>One</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Seven</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Five</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Four</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Three</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nine</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
</ui>
